content/chips/bl808/efuse.md

```diff
@@ -0,0 +1,82 @@
+| ldo_trim_enable      | 22         | 1           | 1             | Apply LDO18 trimming from eFuse (0x78, see `F_Ctrl_Read_LDO18IO_Vout_Trim`                                                                                                                                         |
+| trim_enable          | 23         | 1           | 1             | Apply RC32m trimming from eFuse (0x00, see `F_Ctrl_Read_Xtal_Trim_RC32M`                                                                                                                                           |
+| no_hd_boot_en        | 24         | 1           | 0             |                                                                                                                                                                                                                    |
+| flash_power_delay    | 25         | 2           | 01 (1)        | Time to wait after power-cycling the flash (via `GLB_PU_LDO18FLASH`). 0: none, 1: 1ms, 2: 8ms, 3: 16ms                                                                                                             |
+| tz_boot              | 27         | 1           | 0             | Wide-ranging effects. Disables some bootloader protocol commands (such as WRITE_MEMORY). Disallows ROM-based setup of cores other than M0.                                                                         |
+| encrypted_tz_boot    | 28         | 1           | 0             |                                                                                                                                                                                                                    |
+| hbn_check_sign       | 29         | 1           | 0             |                                                                                                                                                                                                                    |
+| keep_dbg_port_closed | 30         | 1           | 0             | Code at 0x900140b0. Executed before jumping to user code. Sets `TZC_SEC_TZC_SBOOT_DONE` to all-ones.                                                                                                                 |
+| hbn_jump_disable     | 31         | 1           | 0             |                                                                                                                                                                                                                    |
+
+Notes:
+ - Boot ROM debug output is enabled if and only if `uart_log_disable == 1 && uart_log_reopen == 0`. Since `uart_log_disable` is set on all available devboards, burning `uart_log_reopen` enables boot ROM debugging on the UART selected by `bootlog_pin_cfg`.
```

Comment:
`uart_log_reopen` is 0 by default. Didn't you mean that if `uart_log_disable == 1 && uart_log_reopen == 1`

Reply:
Meh, you're right. It should say "is disabled if and only if".

Reply:
Thanks :blush: